iree/task/api.c

```diff
@@ -33,6 +33,15 @@ IREE_FLAG(
     "threads that would otherwise need to perform the syscalls during\n"
     "coordination.");
 
+IREE_FLAG(
+    int32_t, task_worker_local_memory, 64 * 1024,
+    "Specifies the bytes of per-worker local memory allocated for use by\n"
+    "dispatched tiles. Tiles may use less than this but will fail to dispatch\n"
+    "if they require more. Conceptually it is like a stack reservation and\n"
+    "should be treated the same way: the source programs must be built to use\n"
+    "a maximum amount of local memory and the runtime must be configured to\n"
+    "make that amount of local memory available.");
```

Comment:
Can you clarify "the source programs must be built to use a maximum amount of local memory"?

Shouldn't they use a _minimal_ amount of memory? Or is this more "use local instead of shared or global memory as much as possible"?

Reply:
good q - clarified:
"the source programs must be built to only use a specific maximum amount of local memory and the runtime must be configured to make at least that amount of local memory available."
this is similar to stack sizes: you'd compile your program saying "use at most N bytes of stack space" and use no more of that and at runtime specify that same N - but it's ok for the runtime to have a larger N as the program will still run correctly.